app/helpers/moments_helper.rb

```diff
@@ -150,5 +170,24 @@ def data_for(item)
       @moment.strategy
     end
   end
+
+  private
```

Comment:
As Codeclimate identified this is redundant, there's already an existing private modifier in the file, so you'll want to put these new private methods under that.

Reply:
Ahhh! I didn't notice the private modifier on line 18 that already existed. I went ahead and removed the redundant private modifier and moved these new methods to be placed directly underneath the original one.